core/arch/opnd.h

```diff
@@ -1355,7 +1509,7 @@ INSTR_INLINE
  * Returns a register operand corresponding to a part of the
  * register represented by the DR_REG_ constant \p r.
  *
- * On x86, \p r must be a multimedia (mmx, xmm, or ymm) register.  For
+ * On x86, \p r must be a multimedia (mmx, xmm, ymm or zmm) register.  For
```

Comment:
nit: s/ymm/ymm,/ to match comma style throughout

Reply:
Sure